CMakeLists.txt

```diff
@@ -113,9 +113,9 @@ project (Jerry C ASM)
   set(MCU_SCRIPT_GENERATED_HEADER ${CMAKE_BINARY_DIR}/generated.h)
 
 # Should we use external libc?
- if(DEFINED USE_COMPILER_DEFAULT_LIBC AND USE_COMPILER_DEFAULT_LIBC STREQUAL "YES")
+ if(DEFINED COMPILER_DEFAULT_LIBC AND COMPILER_DEFAULT_LIBC STREQUAL "ON")
```

Comment:
Why did you rename this? Are you sure you changed all reference of `USE_COMPILER_DEFAULT_LIBC`?